docs/source/roles/researcher/user_guide_msrds.md

```diff
@@ -2,10 +2,50 @@
 
 # User Guide: Microsoft Remote Desktop
 
-```{include} snippets/01_introduction.partial.md
-:relative-images:
+## {{beginner}} Introduction
+
```

Comment:
If I recall we agreed to add a deprecation warning for MSRDS.

Reply:
Now added a deprecation warning to the user guide and deployment instructions for MSRDS